{
    let a = 2;
    console.log(a); // 2
}
console.log(a); // RefferenceError
/* 
    try{throw 2}catch(a) {
        console.log(a); //2
    }
console.log(a); // Refference Error
*/
